src/Generator/TransformerResolver/ChainTransformerResolver.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AutoMapper\Generator\TransformerResolver;
+
+use AutoMapper\Attribute\PropertyAttribute;
+use AutoMapper\Attribute\MapTo;
+use AutoMapper\CustomTransformer\CustomTransformerGenerator;
+use AutoMapper\CustomTransformer\CustomTransformersRegistry;
+use AutoMapper\Extractor\PropertyMapping;
+use AutoMapper\Transformer\CallbackTransformer;
+use AutoMapper\Transformer\TransformerFactoryInterface;
+use AutoMapper\Transformer\TransformerInterface;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractorInterface;
+
+final readonly class ChainTransformerResolver implements TransformerResolverInterface
```

Comment:
I need to think a little bit more about this one, the inner logic might not stay this way